### ISSUE TITLE
[ADE 508 Audit]: 508 Audit Report for 43785 va.gov Forms - Defect 7 | High

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#107044]
> [!NOTE]
> **QUICK SUMMARY:** _Child ticket to track the work related to Defect 7 | High of https://github.com/department-of-veterans-affairs/va.gov-team/issues/106900 .._

### Defect 7 | High | Created:
_WCAG 4.1.1 Parsing | In content implemented using markup languages, elements have complete start and end tags, elements are nested according to their specifications, elements do not contain duplicate attributes, and any IDs are unique, except where the specifications allow these features._

Throughout the product, when the screen is initially loaded, the names for the radio buttons are read once when reading with the arrow keys but after selecting a radio button, the names are read twice by assistive technology. For example, on the File A Supplemental Claim | Veterans Affairs screen, when reading with the arrow keys, “Disability compensation claim radio button not checked” is read but after selecting this option, “Disability compensation claim radio button checked Disability compensation claim” is read with the arrow keys. (New: 04/04/2025)

Throughout the product, after expanding some buttons, the content cannot be read by assistive technology. For example, on the File A Supplemental Claim | Veterans Affairs screen (Follow these steps to get started), after expanding What happens after I apply?, the content cannot be read by assistive technology. Similarly, on the File A Supplemental Claim | Veterans Affairs screen (Step1 of 4Veteran information), after expanding Are you experiencing domestic violence?, some of the content cannot be read. (New: 04/04/2025)

Throughout the product, some elements are not read correctly by assistive technology. For example, on the File A Supplemental Claim | Veterans Affairs screen (Step1 of 5Veteran information), under Personal information, when tab is used to move the focus to the Find more detailed instructions for how to change your legal name … link, “Find more detailed instructions for how to change your legal name on file (opens in a new tab)opens in a new tab” is read by assistive technology—“opens in a new tab” is read twice. Similarly, on the File A Supplemental Claim | Veterans Affairs screen (Step3 of 5New and relevant evidence), “Review the notice of evidence needed (opens in a new tab)opens in a new tab” is read. (New: 04/04/2025)

Throughout the product, the names for check boxes are read twice when reading with the arrow keys. For example, on the File A Supplemental Claim | Veterans Affairs screen (Step1 of 5Veteran information), “I live or sleep overnight in a place that isn’t meant for regular sleeping. This includes a car, park, abandoned building, bus station, train station, airport, or camping ground. check box not checked I live or sleep overnight in a place that isn’t meant for regular sleeping. This includes a car, park, abandoned building, bus station, train station, airport, or camping ground.” is read. (New: 04/04/2025)

Throughout the product, extra information read after the role does not convey correct information. For example, on the File A Supplemental Claim | Veterans Affairs screen (Step3 of 5New and relevant evidence), when tab is used to move the focus to the check box, “I certify that I have reviewed the notice of evidence needed or I received my decision less than 1 year ago. (*Required) check box not checked If you’re filing a Supplemental Claim more than 1 year after you got your decision notice, you’ll need to review and acknowledge our notice of evidence needed for your disability claim. Review the notice of evidence needed (opens in a new tab) opens in a new tab” is read—this does not open a new tab. On the File A Supplemental Claim | Veterans Affairs screen (Step3 of 5New and relevant evidence), when tab is used to move the focus to the I acknowledge and authorize this release of information (*Required) check box, “… this authorization is signed in Item 11, as w” is read—the text is too long to be read. (New: 04/04/2025)

Feedback from DK:
> Throughout the product, the names for check boxes are read twice when reading with the arrow keys. For example, on the File A Supplemental Claim | Veterans Affairs screen (Step1 of 5Veteran information), “I live or sleep overnight in a place that isn’t meant for regular sleeping. This includes a car, park, abandoned building, bus station, train station, airport, or camping ground. check box not checked I live or sleep overnight in a place that isn’t meant for regular sleeping. This includes a car, park, abandoned building, bus station, train station, airport, or camping ground.” is read. (New: 04/04/2025)

> Throughout the product, when the screen is initially loaded, the names for the radio buttons are read once when reading with the arrow keys but after selecting a radio button, the names are read twice by assistive technology. For example, on the File A Supplemental Claim | Veterans Affairs screen, when reading with the arrow keys, “Disability compensation claim radio button not checked” is read but after selecting this option, “Disability compensation claim radio button checked Disability compensation claim” is read with the arrow keys. (New: 04/04/2025)

Some of this is also called out in the last review on the HLR form. There's a ticket and the issues may be related.

[[Higher Level Review]: Some elements are not announcing correctly in JAWS #106044](https://github.com/department-of-veterans-affairs/va.gov-team/issues/106044)

One of the last things I investigated before the contract ended was this bug in the design system, related to radio buttons. I believe it may be causing the issue called out here with radio buttons and there may be a similar issue with the checkbox component. I ran out of time in looking into it, but want to call it out here to save the new team time.

[VaRadio component selection not changing with arrow keys - 508 audit finding (High severity)](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3928)

Lastly, there are a few callouts to "opens in a new tab" being read out twice. Robin fixed this in the final sprint, with a [PR to the design system component](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3699). So it just needs to get pulled into the codebases for the forms.

Hope this helps!
DK

### Ticket Updates
- [ ] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [ ] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [x] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._

**Comment by mmutia** [comment]
The radio buttons and checkboxes throughout the form are still announcing twice. The label and the hint text are being announced together twice. The checkbox on the "Review and acknowledge the notice of evidence needed." page is additionally announcing twice, despite having no hint text.

**Comment by mmutia** [comment]
Example of component that announces twice:

<input type="checkbox" id="root_contestedIssues_0" name="root_contestedIssues_0" aria-describedby="issue-0-description" aria-labelledby="issue-0-title" data-dd-action-name="Issue Name">

Is it possible that only aria can be used only once?

**Comment by lakper** [comment]
Updates have been made/added to backlogs, closing ticket.
